src/dns_internal.cc

```diff
@@ -833,7 +833,7 @@ idnsDoSendQueryVC(nsvc *vc)
     vc->busy = 1;
 
     // Comm needs seconds but idnsCheckQueue() will check the exact timeout
-    const int timeout = (Config.Timeout.idns_query % 1000 ?
+    const time_t timeout = (Config.Timeout.idns_query % 1000 ?
                          Config.Timeout.idns_query + 1000 : Config.Timeout.idns_query) / 1000;
```

Comment:
Please also consider fixing Ipc::UdsOp::setTimeout() which forwards an `int` timeout argument to commSetConnTimeout().

Reply:
Thanks for the advice. Accepting